docs/state.md

```diff
@@ -9,15 +9,15 @@ While Linux is usable on all T2 models, some features are limited due to the lac
 - USB
 - Keyboard
 - Camera
-- Wifi (For now its working for all the devices we have data about. There are still some devices of whom we don't have data. Refer to the [wifi guide](https://wiki.t2linux.org/guides/wifi/) for more details)
+- Wi-Fi
+- Touch Bar
```

Comment:
Maybe still mention that it's the same as the windows driver, no custom graphics.

Reply:
4d7e402